x/oracle/keeper/msg_server.go

```diff
@@ -227,8 +227,23 @@ func (ms msgServer) GovAddDenoms(
 			return nil, err
 		}
 	}
-	// also update RewardBand key
-	plan.Keys = append(plan.Keys, string(types.KeyRewardBands))
+
+	// append new currency pair providers
+	for _, cpp := range msg.CurrencyPairProviders {
+		plan.Keys = ojoutils.AppendUniqueString(plan.Keys, string(types.KeyCurrencyPairProviders))
```

Comment:
Is there a way to remove currency pair providers with this implementation?

Reply:
[debd18c](https://github.com/ojo-network/ojo/pull/259/commits/debd18c3bb5b8fc06fbc1bc09538372acf59074c)